homeassistant/components/tuya/binary_sensor.py

```diff
@@ -154,6 +154,17 @@ class TuyaBinarySensorEntityDescription(BinarySensorEntityDescription):
             on_value="feeding",
         ),
     ),
+    # Motion Sensor Light
+    # https://developer.tuya.com/en/docs/iot/gyd?id=Kaof8a8hycfmy
+    "gyd": (
+        TuyaBinarySensorEntityDescription(
+            key=DPCode.PIR_STATE,
+            name="Motion",
+            device_class=BinarySensorDeviceClass.MOTION,
+            on_value="pir",
+        ),
```

Comment:
How would this work for a motion sensor? As far as I am aware, motion sensors from Tuya are events, and only send the "on" state, not the "off" state.

Reply:
The Motion sensor within the light sends an off state event based on no motion being detected by the PIR within a defined time, The timings for this are defined in the "PIR delay" entity - which in essence defines the time the Light will remain on after motion has stopped being detected. Time can be defined between 5 seconds and 3600 seconds (1 hour)

Reply:
The problem is: Most Tuya devices don't send that. Hence it has been removed.

Reply:
Do you mean most Tuya devices in the GYD category or generally? The PIR_STATE attribute only seems to be documented against device categories gyd (Motion Sensor Light) and dgnbj (Multi-functional Sensor), the documentation isn't that clear, but does suggest that pir_state can contain one of two values ("pir","none").  The binary sensor does work as noted above against the LEDLIT LTFLPWIFI which I have tested against.

Reply:
> Do you mean most Tuya devices in the GYD category or generally?

Any Pir sensor in Tuya.